pegjs/flinksql.pegjs

```diff
@@ -2986,7 +3006,8 @@ DOUBLE_WELL_ARROW = '#>>'
 
 OPERATOR_CONCATENATION = '||'
 OPERATOR_AND = '&&'
-LOGIC_OPERATOR = OPERATOR_CONCATENATION / OPERATOR_AND
+OPERATOR_XOR = 'XOR'i !ident_start { return 'XOR' }
```

Comment:
I'm not sure that there is an `XOR` in Flink. It's neither a [reserved keyword](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/sql/overview/#reserved-keywords) nor a [system (built-in) function](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/functions/systemfunctions/).
There is a `BIT_XOR()` in Calcite, but it is an [aggregate function](https://calcite.apache.org/docs/reference.html#aggregate-functions).

Reply:
@poulinjulien removed from flink.

Reply:
I should point out, that to my knowledge the XOR operator is only supported by MySQL and MariaDB.